packages/lime-system/tests/test_lime_generic_config.lua

```diff
@@ -0,0 +1,37 @@
+local config = require 'lime.config'
+local gen_cfg = require 'lime.generic_config'
+local test_utils = require 'tests.utils'
+
+local uci = nil
+
+describe('LiMe Generic config tests #genericconfig', function()
+
+    it('test config.do_generic_uci_config', function()
```

Comment:
it would be good to test when a config fails, as it is an important usecase (the first alternate path).

Reply:
Thanks, I am also adding more tests for other paths that werent exercised